plugins/woocommerce/includes/blocks/class-wc-brands-block-template-utils-duplicated.php

```diff
@@ -0,0 +1,367 @@
+<?php // phpcs:disable WordPress.Files.FileName.InvalidClassFileName
+
+declare( strict_types = 1);
+
+use Automattic\WooCommerce\Blocks\Options;
+
+//phpcs:disable Squiz.Classes.ClassFileName.NoMatch
+
+/**
+ * BlockTemplateUtils class used for serving block templates from Woo Blocks.
+ * IMPORTANT: These methods have been duplicated from Gutenberg/lib/full-site-editing/block-templates.php as those functions are not for public usage.
+ *
+ * @version x.x.x
```

Comment:
let's change the version strings, also let's add a note somewhere that these are internal use for Brands code and not meant to be public API.